docs/code_standards.rst

```diff
@@ -59,9 +83,15 @@ Naming
 * _private_variable
 * _PrivateClass
 
+Furthermore, when writing fixtures, make their scope clear at the beginning of the fixture function's name. An unspecified scope is assumed to be function-level
+
+* org
+* module_org
+* session_org
+
 Style
 
-* Absolutely no semi colons.
+* Absolutely no semicolons.
 * Lines are not to exceed 99 characters in length.
 * Code will be indented with **4 spaces** instead of tabs.
```

Comment:
In some editors or IDEs, you can format the indent of your tab to be a certain number of spaces.  Correct me if I'm wrong, but it seems pycharm uses 4 spaces as default for it's tabs.

Reply:
that is correct. for most people, setting this as their editor default is the best behavior. black might also correct for this.